pom.xml

```diff
@@ -231,6 +233,14 @@
 	            <version>${htmlformentryVersion}</version>
 	            <scope>provided</scope>
 	        </dependency>
+
+	        <dependency>
+		        <groupId>org.openmrs.module</groupId>
+		        <artifactId>reporting-api-2.0</artifactId>
+		        <version>${reportingVersion}</version>
+		        <scope>provided</scope>
+	        </dependency>
```

Comment:
You also need to add reporting to required modules in https://github.com/openmrs/openmrs-module-referenceapplication/blob/master/omod/src/main/resources/config.xml
It is to make sure the reporting module is installed and started before the referenceapplication module.